### PR TITLE
chore: Add remote cache in sidecar

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <ce.gao@outlook.com>

### CODE REVIEW

File: docs/.vitepress/config/sidebar.ts [hunk 1]
```diff
@@ -84,7 +87,9 @@ export const sidebar: DefaultTheme.Sidebar = {
       items:[
         { text: 'Faster Build with Software Mirrors',link:'/tips/mirror' },
         { text: 'Per-user Config', link:'/tips/config'},
-        { text: 'CI/CD Integration', link:'/tips/ci'}
+        { text: 'Remote Cache', link: '/tips/cache'},
+        { text: 'CI/CD Integration', link:'/tips/ci'},
+        { text: 'Multi-target Build', link:'/tips/multi-target'}
```
Comment (kemingy) [inline docs/.vitepress/config/sidebar.ts:92]:
```suggestion
        { text: 'Multi-target Build', link:'/tips/multitarget'}
```

File: docs/.vitepress/config/sidebar.ts [hunk 2]
```diff
@@ -27,8 +27,11 @@ export const sidebar: DefaultTheme.Sidebar = {
       collapsible: true,
       items:[
         { text: '利用软件镜像源更快地构建环境',link:'/zh/tips/mirror' },
+        
         { text: '用户相关配置', link:'/zh/tips/config'},
-        { text: 'CI/CD 集成', link:'/zh/tips/ci'}
+        { text: 'Remote Cache', link: '/tips/cache'},
+        { text: 'CI/CD 集成', link:'/zh/tips/ci'},
+        { text: '多目标构建', link:'/tips/multi-target'}
```
Comment (kemingy) [inline docs/.vitepress/config/sidebar.ts:34]: No such file? For cache and multi-target.

**Review by kemingy — CHANGES_REQUESTED** [review]

File: docs/.vitepress/config/sidebar.ts (hunk 2 above)
Comment (gaocegege) [reply docs/.vitepress/config/sidebar.ts:34]: Yep we should use multitarget here.

**Comment by gaocegege** [comment]
Nice catch!

**Review by kemingy — APPROVED** [review]

**Pull request merged**